public/locations-list-map.js

```diff
@@ -507,17 +541,18 @@ function googleMapsUri(address) {
 
 function getEntryEl(entry) {
   if (!entry.domElem) {
-    entry.domElem = ce('div', 'location');
-    ac(entry.domElem, ce('h4', null, ctn(entry.name)));
+    entry.domElem = ce('div', 'location py-3');
+    const header = ce('div', 'd-flex');
+    const headerHospitalInfo = ce('div', 'flex-grow-1');
```

Comment:
I mentioned this in Slack but -- maybe this is an opportunity to replace this logic with handlebars templating?